soh/soh/Enhancements/randomizer/hook_handlers.cpp

```diff
@@ -315,6 +316,34 @@ void RandomizerOnVanillaBehaviorHandler(GIVanillaBehavior id, bool* should, void
             }
             break;
         }
+        case GI_VB_GIVE_ITEM_FROM_COW: {
+            if (!RAND_GET_OPTION(RSK_SHUFFLE_COWS))
+                break;
```

Comment:
Let's always wrap condition scopes with curly brackets. eventually ™️ ci will complain about this automatically